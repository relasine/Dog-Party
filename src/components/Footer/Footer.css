.Footer {
  background-color: #048eaa;
  color: #ffffff;
  padding: 50px 0;
}

h3 {
  font-size: 1.4rem;
  font-weight: 600;
  margin: 0;
  text-align: center;
}

.social-links {
  margin: 30px auto 0 auto;
  width: 300px;
}

.footer-img {
  height: 50px;
  margin: 10px;
}