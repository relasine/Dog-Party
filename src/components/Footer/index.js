import React from 'react';
import './Footer.css';

import twitter from '../../assets/twitter.svg';
import instagram from '../../assets/instagram.svg';
import facebook from '../../assets/facebook.svg';
import linkedin from '../../assets/linkedin.svg';

const Footer = () => {
  const socialLinks = [
  {icon: twitter},
  {icon: instagram},
  {icon: facebook},
  {icon: linkedin}
  ];

  return (
    <h3>Learn About Dogs Online</h3>
  )
}

export default Footer;