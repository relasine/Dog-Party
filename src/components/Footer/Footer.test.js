import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './index.js';
import { shallow, mount } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('should match the snapshot', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
});

it('should have four links', () => {
  const wrapper = mount(<Footer />);

  expect(wrapper.find('a')).toHaveLength(4);
});

it('should have four images', () => {
  const wrapper = mount(<Footer />);

  expect(wrapper.find('img')).toHaveLength(4);
});