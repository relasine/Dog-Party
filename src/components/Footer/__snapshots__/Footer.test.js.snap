// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<footer
  className="Footer"
>
  <h3>
    Learn About Dogs Online
  </h3>
  <div
    className="social-links"
  >
    <a
      href="https://twitter.com/search?q=dogs&src=typd"
      key="Twitterhttps://twitter.com/search?q=dogs&src=typd"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Twitter logo"
        src="twitter.svg"
      />
    </a>
    <a
      href="https://www.instagram.com/dogsofinstagram/"
      key="Instagramhttps://www.instagram.com/dogsofinstagram/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Instagram logo"
        src="instagram.svg"
      />
    </a>
    <a
      href="https://www.facebook.com/BlackDogAnimalRescue/"
      key="Facebookhttps://www.facebook.com/BlackDogAnimalRescue/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Facebook logo"
        src="facebook.svg"
      />
    </a>
    <a
      href="https://www.linkedin.com/company/support-dogs/"
      key="LinkedInhttps://www.linkedin.com/company/support-dogs/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="LinkedIn logo"
        src="linkedin.svg"
      />
    </a>
  </div>
</footer>
`;
